test/testvfdswmr.sh.in

```diff
@@ -1137,6 +1156,17 @@ for options in "-d 1" "-d 1 -F" "-d 2 -l 10" "-d 2 -F -l 10" "-d 1 -t -l 10" "-d
 		continue
 	fi
 	echo launch vfd_swmr_bigset_writer few big, options $options ......may take some time......
+
```

Comment:
Minor: is the indentation for line #1160 to #1169 intentional? or should they be lined up with the rest of the code?

Reply:
Intentional.